tests/generic_config_updater/test_mmu_dynamic_threshold_config_update.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import pytest
+import re
+
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+from tests.common.utilities import wait_until
+from tests.common.helpers.dut_utils import verify_orchagent_running_or_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+pytest.mark.topology('any')
```

Comment:
Currently the test is not being executed in the pipeline. You can add it here(https://github.com/sonic-net/sonic-mgmt/blob/master/.azure-pipelines/pr_test_scripts.yaml)  based on the topology.

Reply:
Done